### ISSUE TITLE
Wrong checksum error but checksum do match

### ISSUE THREAD

**Issue by EsacrihT** [hippie68/gogcheck#3]
Hello,

I'm trying **gogcheck** under **git-bash** 2.39.0.1 (64-bit) + osslsigncode 2.5 (x64-mingw) and it always says "wrong checksum" even if checksum is correct.

![gogcheck_git-bash](https://user-images.githubusercontent.com/59102325/207372941-757c6615-1284-4fec-bf92-1b83ef7786a6.png)

I don't know what is causing this behaviour.

I need a little help with this. :)

**Comment by hippie68** [comment]
Thanks for reporting! The bug should be fixed now in the latest update. Please let me know if you encounter any other issues with Git BASH.

**Comment by EsacrihT** [comment]
Thank you very much, **hippie68**! :))

I don't know if this is a bug or I'm missing some config under Git BASH, but the "Timestamp Server Signature verification" is failing.

![gogcheck_timestamp](https://user-images.githubusercontent.com/59102325/207600170-1adca426-41c5-4e23-bdb9-e6a6e1283c11.png)

I tried specifying a Certificate Authority (CA) file for osslsigncode in the script, but it didn't fix the error.

Thanks for the help.

**Comment by hippie68** [comment]
The correct file should be "/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt". For it to exist it may be necessary to leave "OpenSSL" checked during the setup of Git for Windows.

**Comment by EsacrihT** [comment]
Thanks for the reply, **hippie68**.

I tried changing the certificate line in the script, but it did not cause any change.

In fact, I removed the content from "certfile=" and "osslsigncode_options=" in the script, and the result is the same.
So, it makes me think that the options for osslsigncode are being ignored under Git for Windows.

**- EDIT -**

I switched from Git Bash to **Cygwin** and the later works fine with the digital signature verification.
I installed the osslsigncode packages from Cygwin setup.

![cygog](https://github.com/hippie68/gogcheck/assets/59102325/aa65fe65-4113-4d8d-b6a8-3fab75577026)

:)
